smurfs/src/components/Smurf.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import {banish} from '../actions';
+
+
+
+
+const Smurf = (props) => {
+    const smurf = {
+        name: props.smurf.name,
+        age : props.smurf.age,
+        height: props.smurf.height,
+        id: props.smurf.id
+    }
```

Comment:
Your banish method only requires an id  so this smurf object you are setting is not really being used to any effect